src/QMCWaveFunctions/BsplineFactory/SplineR2R.h

```diff
@@ -71,16 +72,33 @@ class SplineR2R : public BsplineSet
   hContainer_type myH;
   ghContainer_type mygH;
 
+  IndexType m_true_norbs;
+
 public:
   SplineR2R()
   {
-    is_complex = false;
-    className  = "SplineR2R";
-    KeyWord    = "SplineR2R";
+    is_complex   = false;
+    className    = "SplineR2R";
+    KeyWord      = "SplineR2R";
+    m_true_norbs = 0;
   }
 
+  // Sets the number of spline SPOs
+  // Note that this might be < number of splines due to padding
+  void setOrbitalSetSize(int norbs) override { m_true_norbs = norbs; }
+
+  // Gives the number of splined SPOs
+  int getOrbitalSetSize() const override { return m_true_norbs; }
```

Comment:
OrbitalSetSize should be the true size. So this override is not needed. if OrbitalSetSize doesn't give you the size without padding. We should fix OrbitalSetSize.

Reply:
Fixed, see above comment.